AzureMonitorAgent/ama_tst/modules/custom_logs/check_clconf.py

```diff
@@ -0,0 +1,46 @@
+import os
+
+from error_codes import *
+from errors      import error_info
+from helpers     import general_info, geninfo_lookup, run_cmd_output
+
+CLCONF_PATH = "/etc/opt/microsoft/azuremonitoragent/config-cache/fluentbit/td-agent.conf"
+
+def check_customlog_input():
+    cl_input = geninfo_lookup('CL_INPUT')
+    if ( cl_input == None or len(cl_input) == 0):
+        return ERR_CL_INPUT
+    for path in cl_input:
+        check_path = run_cmd_output('ls {0}'.format(path)).strip()
+        if check_path.endswith('No such file or directory'):
```

Comment:
I'd recommend instead checking the result of the command (e.g. the `echo $?` result) or adding a general catch-all in case this doesn't work, in case there's unforeseen errors from the `ls` like insufficient permissions